src/components/MessageCard/MessageCard.css.js

```diff
@@ -271,6 +270,9 @@ export const ActionButtonUI = styled(Button)`
 
 export const ImageUI = styled(Image)`
   border-radius: 3px;
+  height: ${({ height }) => height};
+  width: ${({ width }) => width};
+  max-height: 278px;
```

Comment:
Perhaps you can import the constant from `src/components/MessageCard/MessageCard.jsx:19` and use it here and in line 284, to mitigate the feeling of a "magic number" and if the need to change it arises you only do it on one place.

Reply:
Good catch! thanks :)